src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -643,23 +653,26 @@
           line-height: 1rem;
           vertical-align: middle;
         }
+
+        & + .pagination-description {
+          &:before {
+            @include pagination-separator();
+          }
+        }
       }
 
       &-description {
         white-space: nowrap;
+
+        & + .pagination-list {
```

Comment:
I thought you were missing the case with `.pagination-size` and `.pagination-list` and no `.pagination-description`, but I went back and realized that we actually will display a weird empty separator all the time anyway. So I guess we just don't support the case with no description.

That's fine by me.